app/controllers/api/databases.php

```diff
@@ -1103,27 +1103,35 @@ function updateAttribute(
     ->param('key', '', new Key(), 'Attribute Key.')
     ->param('size', null, new Range(1, APP_DATABASE_ATTRIBUTE_STRING_MAX_LENGTH, Range::TYPE_INTEGER), 'Attribute size for text attributes, in number of characters.')
     ->param('required', null, new Boolean(), 'Is attribute required?')
+    ->param('encrypt', false, new Boolean(), 'Encrypt attribute? Encrypting an attribute means that the attribute can not be queried.', true)
```

Comment:
@gewenyu99 Would love your feedback on the description

Reply:
I suggest giving more context. "Encrypt attribute? [Describe what encrypting an attribute does] Encrypted attributes cannot be queried."

Reply:
@gewenyu99  What about now??

Reply:
> @gewenyu99 What about now??

What does encrypting an attribute do? Can you describe it to me briefly?

Reply:
> > @gewenyu99 What about now??
> 
> What does encrypting an attribute do? Can you describe it to me briefly?

Encrypting an attribute would keep the data encrypted in the database. Incase if a bad actor gets accessed to your data, it wouldn't be able to access the data of the encrypted attribute. Encrypting would also means that we would loose indexing and queries support

Reply:
> Encrypt attribute? Appwrite Database encrypts data in encrypted attributes. Even if malicious actors gain access to your data, they wouldn't be able to access the data of the encrypted attribute. You can't query or index encrypted data.